src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -1335,6 +1371,30 @@ private JSONObject handleMailArchiveMessage(ZipOutputStream out,
 			return messageStatus;
 		}
 
+        private JSONObject handleMailArchiveMboxMessage(ZipOutputStream out,
```

Comment:
No, since the `handleMailArchiveMessage()` takes the email message as JsonObject and the `handleMailArchiveMboxMessage` take email message as string